static/manifest.json

```diff
@@ -1,18 +1,15 @@
 {
+    "name": "PlayCanvas glTF Viewer",
+    "short_name": "glTF Viewer",
+    "description": "Fast and lightweight glTF and gaussian splat PLY viewer.",
+    "display": "standalone",
+    "start_url": "/model-viewer/",
```

Comment:
Isn't `viewer` the preferred URL? We actually support both (without redirecting). I'm thinking we should pick one and redirect from the other.

Reply:
I've always thought `viewer` is just too generic. I also prefer to have the github repo name match the application name match the url. Model-viewer for me strikes the balance between being general enough without being too specific. I will concede though that written out it looks a little clunky, so it's your call :)

While we're at it, the repo's short name above is `glTF Viewer`, which is also not quite correct.

Reply:
Renamed application to "Model Viewer" and using path /viewer/.